<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dell-PC" name="Default test" tests="5" failures="1" timestamp="1 Jul 2020 00:53:09 GMT" time="0.187" errors="0">
  <testcase name="f3" time="0.106" classname="test.Testcase7">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at test.Testcase7.f3(Testcase7.java:17)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- f3 -->
  <testcase name="f" time="0.003" classname="test.Testcase7"/>
  <testcase name="f1" time="0.002" classname="test.Testcase7"/>
  <testcase name="f2" time="0.006" classname="test.Testcase7"/>
  <testcase name="f4" time="0.0" classname="test.Testcase7">
    <skipped/>
  </testcase> <!-- f4 -->
</testsuite> <!-- Default test -->
